### PR TITLE
Seat selection activity implementation

### PR DESCRIPTION
**Pull request by alexaib2002**
Brings seat selection functionality to the application. Created as draft for discussing, as currently it cannot receive info from previous activities.

## TODO
- Must be rebased on top of #20
- ***More changes incoming***

### CODE REVIEW

**Comment by alexaib2002** [comment]
`main` branch was merged instead of rebased, so it improves legibility in the git logs. Conflicts were manually solved (and my `strings.xml` now follow the standards)

File: app/CineHub/app/src/main/java/org/cinehub/javabeans/MovieRoom.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package org.cinehub.javabeans;
```
Comment (Jkutkut) [inline app/CineHub/app/src/main/java/org/cinehub/javabeans/MovieRoom.java]: This javabeans are meant to be merged with the model classes from the API or they should be independent?

Comment (alexaib2002) [reply app/CineHub/app/src/main/java/org/cinehub/javabeans/MovieRoom.java]: They are meant as placeholders, with the arrival of #28 to our main branch they'll be deprecated.

**Comment by alexaib2002** [comment]
- Superseded by #38, check #27 for more details

**Pull request closed without merging**